src/components/LoginCard/index.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { useEffect } from 'react';
+import { useStore } from '../../useStore';
+import { useForm, FieldValues } from 'react-hook-form';
+import { FormInput } from '../FormInputs';
+import Cookies from 'js-cookie';
+import { useNavigate } from 'react-router';
+import { Card, Button, Title, Text, Divider } from '@mantine/core';
+
+export default function LoginCard() {
+  const { logIn, showModal } = useStore();
+  const navigate = useNavigate();
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm();
+
+  async function handleLogin(data: FieldValues) {
+    Cookies.remove('userName');
+    await logIn(data);
+  }
+
+  useEffect(() => {
+    const params = new URLSearchParams(window.location.search);
+
+    if (params.get('error')) {
+      showModal({
+        body: params.get('error'),
+        title: '',
+        okCallback: () => navigate('/login'),
+      });
+    }
+  }, []);
+
+  function loginAsGuest() {
+    if (!localStorage.getItem('DoNotShowGuestAccountWarning')) {
+      showGuestLoginWarning(guestAccountLogin);
+    } else {
+      guestAccountLogin();
+    }
+  }
+
+  function guestAccountLogin() {
+    Cookies.set('userName', 'guest');
+    window.location.href = '/quizzes/guest';
+  }
+
+  function showGuestLoginWarning(okCallback: Function) {
+    showModal({
+      title: 'Logging in as a guest',
+      body: 'Anyone will be able to access the quizzes created in a guest account.',
+      doNotShowAgainKey: 'GuestAccountWarning',
+      okText: 'Continue',
+      okCallback,
+    });
+  }
+
+  return (
+    <Card shadow="xs" p="lg" radius="xs" withBorder className="primaryCard">
+      <Title align="center" order={4}>
+        Log in on Quizmaster
```

Comment:
Make title `Log in` and put logo of the app at the top of the page somewhere outside this card.
Until we decide the logo, we can just use the `Quizmaster` text in some fancy font.
Create a logo component & re-use it wherever needed in sign up etc.